packages/ddp-client/common/livedata_connection.js

```diff
@@ -330,6 +330,7 @@ export class Connection {
   /**
    * @memberOf Meteor
    * @importFromPackage meteor
+   * @alias Meteor.subscribe
```

Comment:
For the sake of posterity...

The alternative to `@alias` I had investigated was the use of [`@borrows`](http://usejsdoc.org/tags-borrows.html) declarations in `ddp-client/client/client_convenience.js`, where the `Meteor.{subscribe,methods,call,apply,status,reconnect,disconnect}` methods are aliased to the connection:

https://github.com/meteor/meteor/blob/9daf412b2b7f69c068319ba3f9762817f210eea1/packages/ddp-client/client/client_convenience.js#L52-L64

In addition to each of those `@borrows` needing a `@namespace Meteor` declaration, they would have had to be partially duplicated in `packages/ddp-server/server_convenience.js`, which would have resulted in duplicate declarations in the resulting `jsdoc` `data/data.json` file, most likely confusing our rather primitive `apibox`es on docs:

https://github.com/meteor/meteor/blob/9daf412b2b7f69c068319ba3f9762817f210eea1/packages/ddp-server/server_convenience.js#L12-L17

Yeah, so I chose the `@alias` method.